src/Standards/Generic/Tests/Formatting/SpaceAfterNotUnitTest.1.inc

```diff
@@ -84,3 +84,6 @@ if ( ! // phpcs:ignore Standard.Cat.SniffName -- for reasons.
 ) {}
 // phpcs:set Generic.Formatting.SpaceAfterNot ignoreNewlines false
 // phpcs:set Generic.Formatting.SpaceAfterNot spacing 1
+
+$var = !	$oneTab;
+$var = !		$twoTabs;
```

Comment:
What's up with these tab based tests ? Unless a sniff has code which is specifically for handling tabs (which this sniff does not have), tests should always be space based.

Tests which do need to test tab handling should [set a meaningful `tabWidth` for the test](https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/63a8cbe65e4ea6a59abcbe5145f98b608f25f816/src/Standards/Generic/Tests/WhiteSpace/DisallowSpaceIndentUnitTest.php#L31-L39) as otherwise the test is useless.

Reply:
I decided to add those two tests to document that the sniff also works with tabs similar to what is done in a few of the `SpaceAfterCast` tests where tabs are also used:

https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/a6c9e2fd626ff63341bbaac0098c8ea0ea11129b/src/Standards/Generic/Tests/Formatting/SpaceAfterCastUnitTest.inc#L56

Reply:
@rodrigoprimo Those tests test that tabs _indentation_ is being removed/left alone in combination with the `ignoreNewlines` property.

The tests in this file do not do that, which means they have no value (other than to confuse, as without a `tabWidth` setting, the fixer doesn't fix these tests correctly).

Reply:
Thanks for the clarification, @jrfnl. I went ahead and amended the original commit to remove those tests.